docs/content/how-to/openstack/perf.md

```diff
@@ -0,0 +1,141 @@
+      namespace: "${HYPERSHIFT_NAMESPACE}"
+    data:
+      tuning: |
+        apiVersion: performance.openshift.io/v2
+        kind: PerformanceProfile
+        metadata:
+          name: cnf-performanceprofile
+        spec:
+          additionalKernelArgs:
+            - nmi_watchdog=0
+            - audit=0
+            - mce=off
+            - processor.max_cstate=1
+            - idle=poll
+            - intel_idle.max_cstate=0
+            - amd_iommu=on
+          cpu:
+            isolated: "${CPU_ISOLATED}"
+            reserved: "${CPU_RESERVED}"
+          hugepages:
+            defaultHugepagesSize: "1G"
+            pages:
+              - count: ${HUGEPAGES}
+                node: 0
+                size: 1G
+          nodeSelector:
+            node-role.kubernetes.io/worker: ''
+          realTimeKernel:
+            enabled: false
+          globallyDisableIrqLoadBalancing: true
+```
+
+Create this PerformanceProfile (change the namespace field in the example above to specify the clusters namespace) by running:
+
+```shell
+oc apply -f perfprof.yaml
+```
+
+## Nodepool creation
+
+After creating the PerformanceProfile, we can create a Nodepool that will use the PerformanceProfile for the nodes.
+
+```shell
+export NODEPOOL_NAME=$CLUSTER_NAME-cnf
+export IMAGE_NAME="rhcos"
+export FLAVOR="m1.xlarge.nfv"
+
+hcp create nodepool openstack \
+  --cluster-name $CLUSTER_NAME \
+  --name $NODEPOOL_NAME \
+  --node-count 0 \
+  --openstack-node-image-name $IMAGE_NAME \
+  --openstack-node-flavor $FLAVOR
+```
+
+Now let's patch the Nodepool to use the PerformanceProfile:
+
+```shell
+oc patch nodepool -n ${HYPERSHIFT_NAMESPACE} ${CLUSTER_NAME} -p '{"spec":{"tuningConfig":[{"name":"cnf-performanceprofile"}]}}' --type=merge
+```
+
+We can scale the Nodepool to the desired number of nodes:
+
+```shell
+oc scale NodePool/$CLUSTER_NAME --namespace ${HYPERSHIFT_NAMESPACE} --replicas=1
+```
+
+To wait for the node to be ready:
+
+```shell
+oc wait --for=condition=UpdatingConfig=True nodepool -n ${HYPERSHIFT_NAMESPACE} ${CLUSTER_NAME} --timeout=5m
+oc wait --for=condition=UpdatingConfig=False nodepool -n ${HYPERSHIFT_NAMESPACE} ${CLUSTER_NAME} --timeout=30m
+oc wait --for=condition=AllNodesHealthy nodepool -n ${HYPERSHIFT_NAMESPACE} ${CLUSTER_NAME} --timeout=5m
+```
+
+Once the nodes have been deployed, you can SSH into the nodes and check the performance settings:
+
+```shell
+ssh core@<node-ip>
+```
+chroot /host
+cat /proc/cmdline
+```
+
+## SR-IOV Network Operator
+
+The SR-IOV Network Operator is a Kubernetes operator that manages SR-IOV devices on the nodes where workloads will be running. In the context of OpenShift on OpenStack, the operator can be used to create SR-IOV network attachments that can be used by workload pods.
+
+The nodes deployed by the Nodepool must be labeled so the SR-IOV Network Operator can be deployed on these nodes.
+
+```shell
+hcp create kubeconfig --name $CLUSTER_NAME > $CLUSTER_NAME-kubeconfig
+```
+
```

Comment:
This looks out of place: you create it and then don't do anything with it. Are you missing an `export`? If so, you should probably add a note about why we're doing this.